ex04.ex

```diff
@@ -55,7 +57,15 @@ defmodule Ex04 do
 
   """
 
-  def min . . . "your code"
+  def min(list) do
```

Comment:
Yup!

This might also be a good place to use single line functions:

~~~ elixir
def min2(a, b) when a > b, do: b
~~~